src/classes/STG_PanelCustomizableRollup_CTRL.cls

```diff
@@ -1,19 +1,165 @@
+    modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
 /**
- * Created by randi.wilson on 12/20/17.
- */
+* @author Salesforce.org
+* @date 2018
+* @group Settings
+* @group-content ../../ApexDocContent/Settings.htm
+* @description Controller extension for the Customizable Rollups settings panel.
+*/
 
-public with sharing class STG_PanelCustomizableRollup_CTRL {
+public with sharing class STG_PanelCustomizableRollup_CTRL extends STG_Panel {
 
-    public static void enableCRLP(){
-        //method to enable CRLP
+    /*********************************************************************************************************
+    * @description Returns the string Id of the Recurring Donations panel.
+    */
+    public override string idPanel() { return 'idPanelCustomizableRollup'; }
+
+    /*******************************************************************************************************
+    * @description is the panel currently Polling for deployment results
+    */
+    public Boolean isPolling {
+        get {
+            if (isPolling == null) {
+                return false;
+            } else {
+                return isPolling;
+            }
+        }
+        private set;
     }
 
-    public PageReference navigate(){
+    /*******************************************************************************************************
+    * @description is the org on the HH Account model
+    */
+    public Boolean isHHAccount {
+        get { return (STG_Panel.stgService.stgCon.npe01__Account_Processor__c == CAO_Constants.HH_ACCOUNT_PROCESSOR); }
```

Comment:
I think stylistically we want curly brackets to always open a new indented line, even when there's only a single line inside the brackets.